app/models/warehouse/case_report.rb

```diff
@@ -186,6 +186,10 @@ def process_offender_sar(kase, case_report)
         case_report.third_party_company_name = kase.third_party_company_name
         case_report.number_of_exempt_pages = kase.number_exempt_pages
         case_report.number_of_final_pages = kase.number_final_pages
+        case_report.number_of_days_for_vetting = kase.number_of_days_for_vetting
+        user_for_vetting = kase.user_dealing_with_vetting
+        case_report.user_dealing_with_vetting = user_for_vetting.nil? ? nil : user_for_vetting.full_name
```

Comment:
I think you can call `user_for_vetting&.full_name` and `user_for_vetting&.id` here (safe navigation operator, see: https://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/)

As the method returns `nil` and `nil&.full_name` ==> `nil`

Reply:
didn't know this,  nice, I changed to the above one